docker/Readme.md

```diff
@@ -0,0 +1,41 @@
+# Gen Containerization
```

Comment:
Perhaps rather than in its own markdown file, this should be added as a subsection at the bottom of `docs/src/getting_started.md`.  I say bottom because I think Gen wants to present primarily as an ordinary Julia package rather than a monolithic stack (perhaps @marcoct can comment).  Also, we should include in the beginning of this section an explanation of why the user might ever want to use these container instructions (e.g., for portability / saves the need to install anything other than Docker).